front/src/reducers/user/index.ts

```diff
@@ -39,12 +44,25 @@ export const userSlice = createSlice({
     logoutSuccess() {
       return userInitialState;
     },
+    setUserRoles(state, action: PayloadAction<{ userRoles: BuiltinRole[] | undefined }>) {
+      state.userRoles = action.payload.userRoles ?? [];
+    },
     updateUserPreferences(state, action: PayloadAction<{ safeWord: string }>) {
       state.userPreferences = action.payload;
     },
+    switchStdcmV2Activated(state) {
```

Comment:
This seems unused?

Reply:
It has been removed by Sarah in this PR: https://github.com/OpenRailAssociation/osrd/pull/9325